### ISSUE TITLE
Will scripts be added to future windows versions?

### ISSUE THREAD

**Issue by ltmoxi** [ProxymanApp/proxyman-windows-linux#127]
## Description
![image](https://user-images.githubusercontent.com/39732676/198244129-aa84a043-507e-45eb-aa65-cb2464a7bcda.png)
This is very useful on mac, but the windows version does not have this function, I hope to add this function.

**Comment by NghiaTranUIT** [comment]
Yes, we definitely port all debugging tools from the Mac version to Windows 👍 

Please stay tuned 🙌
